Modules/Scripted/EditorLib/LabelSelectDialog.py

```diff
@@ -0,0 +1,94 @@
+
+import qt
+import slicer
+
+#-----------------------------------------------------------------------------
+def _map_property(objfunc, name):
```

Comment:
> It seems the function is not specific to QObject, it allows to create a property associated with the attribute of an other object.

Yes, using `getattr` and `setattr` should work with any Python entity which supports those in a reasonable manner. (Actually, I guess in that sense, "attributename" would in fact be reasonable. I just used it for QObject properties.) In which case the above should be fine replacing "QObject" with "object" (don't forget the "a" → "an" :grin:).

Nit: capitalize "Python"?